src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer/events.jelly

```diff
@@ -24,7 +24,7 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout title="${it.displayName} log" permission="${app.ADMINISTER}">
+  <l:layout title="${it.displayName} log" permission="${app.MANAGE}">
```

Comment:
Unrelated to PR title I guess, along with `kubectl` wrapper and other stuff.

Reply:
i've updated the PR title know that i have a better understanding this plugin in general